src/UI/Nav.elm

```diff
@@ -0,0 +1,77 @@
+module UI.Nav exposing (..)
+
+import Element exposing (..)
+import Element.Background as EltBackground
+import Element.Events exposing (onClick)
+import Element.Font exposing (..)
+import Route
+import Session exposing (Session, isGuest)
+import UI.UI as UI
+
+
+type State
+    = State Bool
```

Comment:
The boolean describes if the navigation is open or close

Reply:
Seems reasonable for now. If we need it to be more semantic later we can always refactor to create a custom type `Open|Closed`. 💭